modules/nodes-descheduler-configuring-strategies.adoc

```diff
@@ -32,24 +32,31 @@ spec:
   strategies:
     - name: "LowNodeUtilization" <1>
       params:
-       - name: "cputhreshold"
+       - name: "CPUThreshold"
          value: "10"
-       - name: "memorythreshold"
+       - name: "MemoryThreshold"
          value: "20"
-       - name: "podsthreshold"
+       - name: "PodsThreshold"
          value: "30"
-       - name: "memorytargetthreshold"
+       - name: "MemoryTargetThreshold"
          value: "40"
-       - name: "cputargetthreshold"
+       - name: "CPUTargetThreshold"
          value: "50"
-       - name: "podstargetthreshold"
+       - name: "PodsTargetThreshold"
          value: "60"
-       - name: "nodes"
+       - name: "NumberOfNodes"
          value: "3"
     - name: "RemoveDuplicates" <2>
+    - name: "RemovePodsHavingTooManyRestarts" <3>
+      params:
+       - name: "PodRestartThreshold"
+         value: "10"
+       - name: "IncludeInitContainers"
```

Comment:
@bergerhoffer sorry about this, but I just noticed that we had this param documented with a typo in our repo, it should be `IncludingInitContainers` (see https://github.com/openshift/cluster-kube-descheduler-operator/pull/110)

Reply:
@damemi No problem, glad you caught it. I've updated if you want to double check, thanks!